database/db_user_test.go

```diff
@@ -57,13 +56,13 @@ func TestGormDBUpdateAccessToken(t *testing.T) {
 	if err := db.UpdateAccessToken(updateAccessToken); err != nil {
 		t.Error(err)
 	}
-	updatedUser, err := db.GetUser(user.ID)
+	gotUser, err := db.GetUser(user.ID)
 	if err != nil {
 		t.Fatal(err)
 	}
-	updatedUser.Enrollments = nil
-	if !reflect.DeepEqual(updatedUser, wantUser) {
-		t.Errorf("have user %+v want %+v", updatedUser, wantUser)
+	gotUser.Enrollments = nil
+	if diff := cmp.Diff(wantUser, gotUser, protocmp.Transform()); diff != "" {
+		t.Errorf("GetUser() mismatch (-wantUser, +gotUser):n%s", diff)
```

Comment:
Looks like all your `t.Errorf` calls is missing the slash in newline character as the end; it should be `:\n%s`

Reply:
A great way to check your test cases is to force them to fail so that you can inspect the output; you can simply modify one of the fields being checked.